app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -46,7 +46,7 @@ register_host() {
 <%= "       --data 'host[operatingsystem_id]=#{@operatingsystem.id}' \\\n" if @operatingsystem -%>
 <%= "       --data 'setup_insights=#{@setup_insights}' \\\n" unless @setup_insights.nil? -%>
 <%= "       --data 'setup_remote_execution=#{@setup_remote_execution}' \\\n" unless @setup_remote_execution.nil? -%>
-<%= "       --data 'remote_execution_interface=#{@remote_execution_interface}' \\\n" if @remote_execution_interface.present? -%>
+<%= "       --data 'remote_execution_interface=#{@remote_execution_interface}' \\\n" if @remote_execution_interface.present? -%> 2> /dev/null
```

Comment:
Why are we still silencing the errors? :thinking:

Reply:
See https://github.com/theforeman/foreman/pull/8332#discussion_r580092605.